dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -329,7 +329,27 @@ private static void updateHTMLPageVersionTS(final String id) throws DotDataExcep
      * 
      * @param pageIdentifier The HTMLPage Identifier to pass in
      * @throws DotContentletStateException
+<<<<<<< HEAD
      * @throws DotDataException
+=======
+     * @throws DotDataException 
+     * @throws DotSecurityException 
+     *            
+     */
+	private static void updateVersionTs(String id, Long languageId) throws DotDataException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "updateVersionTs" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)